cmake/dds_idl_sources.cmake

```diff
@@ -135,7 +135,7 @@ function(opendds_target_idl_sources target)
     unset(_ddsidl_cmd_arg_-o)
     unset(_ddsidl_cmd_arg_-Wb,java)
 
-    cmake_parse_arguments(_ddsidl_cmd_arg "-SI;-GfaceTS;-Wb,java" "-o" "" ${_ddsidl_flags})
+    cmake_parse_arguments(_ddsidl_cmd_arg "-SI;-GfaceTS;-Wb,java --default-nested" "-o" "" ${_ddsidl_flags})
```

Comment:
@ceneblock Looking at this again, I'm not sure what this line is actually doing. I know it doesn't work though, because in Azure the call to opendds_idl in the cmake test doesn't include `--default-nested`.

Reply:
Yeah, still not sure and I don't what to get off track on this so I'm just reverting back to what Mason wrote. I'm setting `--default-nested` in `OpenDDSConfig.cmake` where it can be overridden by an option.